geode-apis-compatible-with-redis/src/integrationTest/java/org/apache/geode/redis/internal/executor/connection/AbstractAuthIntegrationTest.java

```diff
@@ -0,0 +1,183 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more contributor license
+ * agreements. See the NOTICE file distributed with this work for additional information regarding
+ * copyright ownership. The ASF licenses this file to You under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance with the License. You may obtain a
+ * copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software distributed under the License
+ * is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+ * or implied. See the License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package org.apache.geode.redis.internal.executor.connection;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.assertThatThrownBy;
+
+import io.lettuce.core.RedisClient;
+import io.lettuce.core.RedisURI;
+import org.junit.Test;
+import redis.clients.jedis.Jedis;
+import redis.clients.jedis.Protocol;
+
+
+public abstract class AbstractAuthIntegrationTest {
+
+  static final String USERNAME = "default";
+  // Since we're going to use a SimpleSecurityManager where password == username means success
+  static final String PASSWORD = USERNAME;
+  Jedis jedis;
+
+  protected abstract void setupCacheWithSecurity(boolean withSecurityManager) throws Exception;
```

Comment:
The value passed for `withSecurityManager` is always `true`. Do we need this method to take an argument if we also have the `setupCacheWithoutSecurity()` method? Alternately, are tests needed for the case where `withSecurityManager` is false?

Reply:
I've removed the parameter.